"use strict";(self.webpackChunkdsong_lab=self.webpackChunkdsong_lab||[]).push([[707],{4161:(e,o,r)=>{r.r(o),r.d(o,{comp:()=>i,data:()=>a});var n=r(6254);const t={},i=(0,r(6995).A)(t,[["render",function(e,o){return(0,n.uX)(),(0,n.CE)("div",null,o[0]||(o[0]=[(0,n.Fv)('<h2 id="preprint" tabindex="-1"><a class="header-anchor" href="#preprint"><span>Preprint</span></a></h2><ol><li><p>Hemminger, Sanchez-Tam, De Ocampo, Wang, Underwood, Xie, Zhao, <strong>Song</strong>, Li, Dong, and Wollman.<br> &quot;Spatial Single-Cell Mapping of Transcriptional Differences Across Genetic Backgrounds in Mouse Brains.&quot;<br><em>bioRxiv</em> (2024).<br> DOI: <a href="https://doi.org/10.1101/2024.10.08.617260" target="_blank" rel="noopener noreferrer">10.1101/2024.10.08.617260</a></p></li><li><p>Li, M.Patel, <strong>Song</strong>, Yan, Li, and Pinello.<br> &quot;Benchmarking computational methods to identify spatially variable genes and peaks.&quot;<br><em>bioRxiv</em> (2023).<br> DOI: <a href="https://doi.org/10.1101/2023.12.02.569717" target="_blank" rel="noopener noreferrer">10.1101/2023.12.02.569717</a></p></li><li><p>Song, Liu, Dai, McMyn, Wang, Yang, Krejci, Vasilyev, Untermoser, Loregger, <strong>Song</strong>, Williams, Rosen, Cheng, Chao, Kale, Zhang, Diao, Bürckstümmer, Siliciano, Li, Siliciano, Huangfu, and Li.<br> &quot;Decoding Heterogenous Single-cell Perturbation Responses.&quot;<br><em>bioRxiv</em> (2023).<br> DOI: <a href="https://doi.org/10.1101/2023.10.30.564796" target="_blank" rel="noopener noreferrer">10.1101/2023.10.30.564796</a></p></li><li><p>Wang, Zhai, <strong>Song</strong>, and Li.<br> &quot;Review of computational methods for estimating cell potency from single-cell RNA-seq data, with a detailed analysis of discrepancies between method description and code implementation.&quot;<br><em>arXiv</em> (2023).<br> DOI: <a href="https://doi.org/10.48550/arXiv.2309.13518" target="_blank" rel="noopener noreferrer">10.48550/arXiv.2309.13518</a></p></li><li><p><strong>Song</strong>, Li, Ge, and Li.<br> &quot;ClusterDE: a post-clustering differential expression (DE) method robust to false-positive inflation caused by double dipping.&quot;<br><em>bioRxiv</em> (2023).<br> DOI: <a href="https://doi.org/10.1101/2023.07.21.550107" target="_blank" rel="noopener noreferrer">10.1101/2023.07.21.550107</a></p></li></ol><h2 id="peer-reviewed" tabindex="-1"><a class="header-anchor" href="#peer-reviewed"><span>Peer reviewed</span></a></h2><ol><li><p>Kock, Kimes, Gisselbrecht, Inukai, Phanor, Anderson, Ramakrishnan, Lipper, <strong>Song</strong>, Kurland, Rogers, Jeong, Blacklow, Irizarry, and Bulyk.<br> &quot;DNA binding analysis of rare variants in homeodomains reveals homeodomain specificity-determining residues.&quot;<br><em>Nature Communications</em> (2024).<br> DOI: <a href="https://doi.org/10.1038/s41467-024-47396-0" target="_blank" rel="noopener noreferrer">10.1038/s41467-024-47396-0</a></p></li><li><p><strong>Song</strong>, Wang, Yan, Liu, Sun, and Li.<br> &quot;scDesign3 generates realistic in silico data for multimodal single-cell and spatial omics.&quot;<br><em>Nature Biotechnology</em> (2024).<br> DOI: <a href="https://doi.org/10.1038/s41587-023-01772-1" target="_blank" rel="noopener noreferrer">10.1038/s41587-023-01772-1</a></p></li><li><p>Yan, <strong>Song</strong>, and Li.<br> &quot;scReadSim: a single-cell RNA-seq and ATAC-seq read simulator.&quot;<br><em>Nature Communications</em> (2023).<br> DOI: <a href="https://doi.org/10.1038/s41467-023-43162-w" target="_blank" rel="noopener noreferrer">10.1038/s41467-023-43162-w</a></p></li><li><p>Cui, <strong>Song</strong>, Wong, and Li.<br> &quot;Single-cell generalized trend model (scGTM): a flexible and interpretable model of gene expression trend along cell pseudotime.&quot;<br><em>Bioinformatics</em> (2022).<br> DOI: <a href="https://doi.org/10.1093/bioinformatics/btac423" target="_blank" rel="noopener noreferrer">10.1093/bioinformatics/btac423</a></p></li><li><p>Jiang, Sun, <strong>Song</strong>, and Li.<br> &quot;Statistics or biology: the zero-inflation controversy about scRNA-seq data.&quot;<br><em>Genome Biology</em> (2022).<br> DOI: <a href="https://doi.org/10.1186/s13059-022-02601-5" target="_blank" rel="noopener noreferrer">10.1186/s13059-022-02601-5</a></p></li><li><p><strong>Song</strong>, Xi, Li, and Wang.<br> &quot;scSampler: fast diversity-preserving subsampling of large-scale single-cell transcriptomic data.&quot;<br><em>Bioinformatics</em> (2022).<br> DOI: <a href="https://doi.org/10.1093/bioinformatics/btac271" target="_blank" rel="noopener noreferrer">10.1093/bioinformatics/btac271</a></p></li><li><p>Sun, <strong>Song</strong>, Li, and Li.<br> &quot;Simulating Single-Cell Gene Expression Count Data with Preserved Gene Correlations by scDesign2.&quot;<br><em>Journal of Computational Biology</em> (2022).<br> DOI: <a href="https://doi.org/10.1089/cmb.2021.0440" target="_blank" rel="noopener noreferrer">10.1089/cmb.2021.0440</a></p></li><li><p>Ge, Chen, <strong>Song</strong>, McDermott, Woyshner, Manousopoulou, Wang, Li, Wang, and Li.<br> &quot;Clipper: p-value-free FDR control on high-throughput data from two conditions.&quot;<br><em>Genome Biology</em> (2021).<br> DOI: <a href="https://doi.org/10.1186/s13059-021-02506-9" target="_blank" rel="noopener noreferrer">10.1186/s13059-021-02506-9</a></p></li><li><p><strong>Song</strong>, Li, Hemminger, Wollman, and Li.<br> &quot;scPNMF: sparse gene encoding of single cells to facilitate gene selection for targeted gene profiling.&quot;<br><em>Bioinformatics</em> (2021).<br> DOI: <a href="https://doi.org/10.1093/bioinformatics/btab273" target="_blank" rel="noopener noreferrer">10.1093/bioinformatics/btab273</a></p></li><li><p>Sun, <strong>Song</strong>, Li, and Li.<br> &quot;scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured.&quot;<br><em>Genome Biology</em> (2021).<br> DOI: <a href="https://doi.org/10.1186/s13059-021-02367-2" target="_blank" rel="noopener noreferrer">10.1186/s13059-021-02367-2</a></p></li><li><p><strong>Song</strong> and Li.<br> &quot;PseudotimeDE: inference of differential gene expression along cell pseudotime with well-calibrated p-values from single-cell RNA sequencing data.&quot;<br><em>Genome Biology</em> (2021).<br> DOI: <a href="https://doi.org/10.1186/s13059-021-02341-y" target="_blank" rel="noopener noreferrer">10.1186/s13059-021-02341-y</a></p></li><li><p>Miller, Hayashi, <strong>Song</strong>, and Wiens.<br> &quot;Explaining the ocean&#39;s richest biodiversity hotspot and global patterns of fish diversity.&quot;<br><em>Proceedings of the Royal Society B: Biological Sciences</em> (2018).<br> DOI: <a href="https://doi.org/10.1098/rspb.2018.1314" target="_blank" rel="noopener noreferrer">10.1098/rspb.2018.1314</a></p></li><li><p><strong>Song</strong>, Wang, Song, Zhou, Xu, Yang, Yang, and Lu.<br> &quot;Increased novel single nucleotide polymorphisms in weedy rice populations associated with the change of farming styles: Implications in adaptive mutation and evolution.&quot;<br><em>Journal of Systematics and Evolution</em> (2017).<br> DOI: <a href="https://doi.org/10.1111/jse.12230" target="_blank" rel="noopener noreferrer">10.1111/jse.12230</a></p></li></ol>',4)]))}]]),a=JSON.parse('{"path":"/publications.html","title":"Publications","lang":"en-US","frontmatter":{"title":"Publications","description":"Preprint Hemminger, Sanchez-Tam, De Ocampo, Wang, Underwood, Xie, Zhao, Song, Li, Dong, and Wollman. \\"Spatial Single-Cell Mapping of Transcriptional Differences Across Genetic B...","head":[["meta",{"property":"og:url","content":"https://vuepress-theme-hope-docs-demo.netlify.app/dsong-lab.github.io/publications.html"}],["meta",{"property":"og:site_name","content":"DSong Lab"}],["meta",{"property":"og:title","content":"Publications"}],["meta",{"property":"og:description","content":"Preprint Hemminger, Sanchez-Tam, De Ocampo, Wang, Underwood, Xie, Zhao, Song, Li, Dong, and Wollman. \\"Spatial Single-Cell Mapping of Transcriptional Differences Across Genetic B..."}],["meta",{"property":"og:type","content":"article"}],["meta",{"property":"og:locale","content":"en-US"}],["meta",{"property":"og:updated_time","content":"2025-03-05T15:41:45.000Z"}],["meta",{"property":"article:modified_time","content":"2025-03-05T15:41:45.000Z"}],["script",{"type":"application/ld+json"},"{\\"@context\\":\\"https://schema.org\\",\\"@type\\":\\"Article\\",\\"headline\\":\\"Publications\\",\\"image\\":[\\"\\"],\\"dateModified\\":\\"2025-03-05T15:41:45.000Z\\",\\"author\\":[{\\"@type\\":\\"Person\\",\\"name\\":\\"Dongyuan Song\\",\\"url\\":\\"https://mister-hope.com\\"}]}"]]},"git":{"createdTime":1741189305000,"updatedTime":1741189305000,"contributors":[{"name":"CZHanoi","username":"CZHanoi","email":"21301010003@m.fudan.edu.cn","commits":1,"url":"https://github.com/CZHanoi"}]},"readingTime":{"minutes":2.06,"words":618},"filePathRelative":"publications.md","localizedDate":"March 5, 2025","autoDesc":true}')},6995:(e,o)=>{o.A=(e,o)=>{const r=e.__vccOpts||e;for(const[e,n]of o)r[e]=n;return r}}}]);