"use strict";(self.webpackChunkdsong_lab=self.webpackChunkdsong_lab||[]).push([[201],{1934:(e,t,i)=>{i.r(t),i.d(t,{comp:()=>n,data:()=>s});var a=i(6254);const o={},n=(0,i(6995).A)(o,[["render",function(e,t){return(0,a.uX)(),(0,a.CE)("div",null,t[0]||(t[0]=[(0,a.Fv)('<p>Our lab studies various problems of data science in genomics, combining <em>bioinformatics</em>, <em>statistics</em>, and <em>machine learning</em>. We develop computational tools for analyzing &quot;omics&quot; data. Currently, our lab focuses on single-cell transcriptomics (gene expression within individual cells) and spatial transcriptomics (gene expression across spatial locations). One central goal is to better understand the cellular heterogenuity and distinguish the biological signals from techinical variation. Several ongoing research directions include:</p><h3 id="omics-data-simulation" tabindex="-1"><a class="header-anchor" href="#omics-data-simulation"><span>Omics data simulation</span></a></h3><div style="display:flex;align-items:center;"><div style="flex:1;"><p><em>How can we generate realistic omics data from computers?</em> The short answer is building a model that learns from the real data. The realistic simulation of single-cell and spatial multi-omics data plays a critical role in both evaluating <strong>performance of computational tools</strong> and facilitating the exploration of <strong>experimental designs</strong>. We developed <a href="https://doi.org/10.1038/s41587-023-01772-1" target="_blank" rel="noopener noreferrer">scDesign3 (Nature Biotechnology, 2024)</a>, the first an &quot;all-in-one&quot; multimodal single-cell and spatial omics simulator which uses a parametric model. I also contributed to the development of <a href="https://doi.org/10.1038/s41467-023-43162-w" target="_blank" rel="noopener noreferrer">scReadSim (Nature Communications, 2023)</a>, a single-cell RNA-seq and ATAC-seq sequencing read read simulator, and <a href="https://doi.org/10.1186/s13059-021-02367-2" target="_blank" rel="noopener noreferrer">scDesign2 (Genome Biology, 2021)</a>, the predecessor of scDesign3.</p></div><div style="margin-left:10px;"><img src="https://dsong-lab.github.io/images/research/scDesign3_example.png" alt="Description" width="300px"></div></div><h3 id="statistical-testing-of-gene-expression-and-co-expression" tabindex="-1"><a class="header-anchor" href="#statistical-testing-of-gene-expression-and-co-expression"><span>Statistical testing of gene expression and co-expression</span></a></h3><div style="display:flex;align-items:center;"><div style="flex:1;"><p><em>How can we accurately detect the changes in gene expression level and co-expression strengh?</em> We need to consider all types if variation and randomess for a solid conclusion. For example, in scRNA-seq, the cell types are usually annotated by clustering, then the differential expression (DE) test between clusters will find many fake DE genes (<strong>double dipping</strong>). We developed <a href="https://www.biorxiv.org/content/10.1101/2023.07.21.550107v1" target="_blank" rel="noopener noreferrer">ClusterDE (Accepted by RECOMB2025)</a>, a post-clustering DE test method controlling false discovery rate (FDR) under double dipping. Another exmample is the trajectory (pseudotime) inference; people usually ignore the fact that pseudotime is <strong>random</strong>. I developed <a href="https://doi.org/10.1186/s13059-021-02341-y" target="_blank" rel="noopener noreferrer">PseudotimeDE (Genome Biology, 2021)</a>, a DE test method for checking gene changes along cell pseudotime accounting for the uncertainty of pseudotime. In addition, we developed <a href="https://doi.org/10.1093/bioinformatics/btac423" target="_blank" rel="noopener noreferrer">scGTM (Bioinformatics, 2022)</a>, an interpretable statistical model for capturing <strong>a gene&#39;s expression trend</strong>.<br> Recently, we are developing a novel co-expression modeling method, named <a href="https://github.com/chexjiang/spCorr" target="_blank" rel="noopener noreferrer">spCorr</a>, for detecting the <strong>dynamic co-expression changes across different locations</strong> in spatial transcriptomics. By using a novel regression model, spCorr is very flexible and can deal with various confounding effects.</p></div><div style="margin-left:10px;"><img src="https://dsong-lab.github.io/images/research/ClusterDE_cartoon.png" alt="Description" width="300px"></div></div><h3 id="reducing-the-complexity-of-large-scale-omics-data" tabindex="-1"><a class="header-anchor" href="#reducing-the-complexity-of-large-scale-omics-data"><span>Reducing the complexity of large-scale omics data</span></a></h3><div style="display:flex;align-items:center;"><div style="flex:1;"><p><em>How can we reduce the complexity of current datasets with numerous genes and millions of cells?</em> In the era of big data, one simple solution is selecting the <strong>most imformative (or representitive) genes and cells</strong>. We developed <a href="https://doi.org/10.1093/bioinformatics/btab273" target="_blank" rel="noopener noreferrer">scPNMF (Bioinformatics; ISMB/ECCB, 2021)</a>, a gene selection method by using dimentionality reduction (Projective NMF) to get a sparse encoding of the expression matrix. In addition, we developed <a href="https://doi.org/10.1093/bioinformatics/btac271" target="_blank" rel="noopener noreferrer">scSampler (Bioinformatics, 2022)</a>, a diveristy-preserving cell subsampling method for large-scale datasets.</p></div><div style="margin-left:10px;"><img src="https://dsong-lab.github.io/images/research/scSampler_example.png" alt="Description" width="300px"></div></div>',7)]))}]]),s=JSON.parse('{"path":"/research.html","title":"Research","lang":"en-US","frontmatter":{"title":"Research","description":"Our lab studies various problems of data science in genomics, combining bioinformatics, statistics, and machine learning. We develop computational tools for analyzing \\"omics\\" da...","head":[["meta",{"property":"og:url","content":"https://vuepress-theme-hope-docs-demo.netlify.app/dsong-lab.github.io/research.html"}],["meta",{"property":"og:site_name","content":"DSong Lab"}],["meta",{"property":"og:title","content":"Research"}],["meta",{"property":"og:description","content":"Our lab studies various problems of data science in genomics, combining bioinformatics, statistics, and machine learning. We develop computational tools for analyzing \\"omics\\" da..."}],["meta",{"property":"og:type","content":"article"}],["meta",{"property":"og:locale","content":"en-US"}],["meta",{"property":"og:updated_time","content":"2025-03-05T15:41:45.000Z"}],["meta",{"property":"article:modified_time","content":"2025-03-05T15:41:45.000Z"}],["script",{"type":"application/ld+json"},"{\\"@context\\":\\"https://schema.org\\",\\"@type\\":\\"Article\\",\\"headline\\":\\"Research\\",\\"image\\":[\\"\\"],\\"dateModified\\":\\"2025-03-05T15:41:45.000Z\\",\\"author\\":[{\\"@type\\":\\"Person\\",\\"name\\":\\"Dongyuan Song\\",\\"url\\":\\"https://mister-hope.com\\"}]}"]]},"git":{"createdTime":1741189305000,"updatedTime":1741189305000,"contributors":[{"name":"CZHanoi","username":"CZHanoi","email":"21301010003@m.fudan.edu.cn","commits":1,"url":"https://github.com/CZHanoi"}]},"readingTime":{"minutes":1.9,"words":571},"filePathRelative":"research.md","localizedDate":"March 5, 2025","autoDesc":true}')},6995:(e,t)=>{t.A=(e,t)=>{const i=e.__vccOpts||e;for(const[e,a]of t)i[e]=a;return i}}}]);