<!doctype html>
<html lang="en-US" data-theme="light">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width,initial-scale=1" />
    <meta name="generator" content="VuePress 2.0.0-rc.20" />
    <meta name="theme" content="VuePress Theme Hope 2.0.0-rc.73" />
    <style>
      :root {
        --vp-c-bg: #fff;
      }

      [data-theme="dark"] {
        --vp-c-bg: #1b1b1f;
      }

      html,
      body {
        background: var(--vp-c-bg);
      }
    </style>
    <script>
      const userMode = localStorage.getItem("vuepress-theme-hope-scheme");
      const systemDarkMode =
        window.matchMedia &&
        window.matchMedia("(prefers-color-scheme: dark)").matches;

      if (userMode === "dark" || (userMode !== "light" && systemDarkMode)) {
        document.documentElement.setAttribute("data-theme", "dark");
      }
    </script>
    <meta property="og:url" content="https://vuepress-theme-hope-docs-demo.netlify.app/dsong-lab.github.io/research.html"><meta property="og:site_name" content="DSong Lab"><meta property="og:title" content="Research"><meta property="og:description" content="Our lab studies various problems of data science in genomics, combining bioinformatics, statistics, and machine learning. We develop computational tools for analyzing "omics" da..."><meta property="og:type" content="article"><meta property="og:locale" content="en-US"><meta property="og:updated_time" content="2025-03-05T15:41:45.000Z"><meta property="article:modified_time" content="2025-03-05T15:41:45.000Z"><script type="application/ld+json">{"@context":"https://schema.org","@type":"Article","headline":"Research","image":[""],"dateModified":"2025-03-05T15:41:45.000Z","author":[{"@type":"Person","name":"Dongyuan Song","url":"https://mister-hope.com"}]}</script><title>Research | DSong Lab</title><meta name="description" content="Our lab studies various problems of data science in genomics, combining bioinformatics, statistics, and machine learning. We develop computational tools for analyzing "omics" da...">
    <link rel="stylesheet" href="/dsong-lab.github.io/assets/css/styles.ceb8c649.css">
    <link rel="preload" href="/dsong-lab.github.io/assets/js/runtime~app.267cafa3.js" as="script"><link rel="preload" href="/dsong-lab.github.io/assets/css/styles.ceb8c649.css" as="style"><link rel="preload" href="/dsong-lab.github.io/assets/js/780.a6b3df29.js" as="script"><link rel="preload" href="/dsong-lab.github.io/assets/js/app.9e8e9631.js" as="script">
    <link rel="prefetch" href="/dsong-lab.github.io/assets/js/13.c1f322ce.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/demo_markdown.html.d0e3c07b.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/publications.html.7660323c.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/research.html.b2a9f76a.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/demo_page.html.6d862ee8.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/Team.html.e6251222.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/demo_layout.html.2e1e3657.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/portfolio.html.523d798c.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/demo_encrypt.html.e19e4572.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/demo_disable.html.ef016a5d.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/Team_index.html.519d66e1.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/518.e136ec34.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/demo_index.html.4cc731b6.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/news.html.248a6dd9.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/index.html.4c428f1a.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/js/404.html.a0be1a65.js" as="script"><link rel="prefetch" href="/dsong-lab.github.io/assets/css/944.styles.e4d3906b.css" as="style">
  </head>
  <body>
    <div id="app"><!--[--><!--[--><!--[--><span tabindex="-1"></span><a href="#main-content" class="vp-skip-link sr-only">Skip to main content</a><!--]--><!--[--><div class="theme-container no-sidebar external-link-icon has-toc" vp-container><!--[--><header id="navbar" class="vp-navbar" vp-navbar><div class="vp-navbar-start"><button type="button" class="vp-toggle-sidebar-button" title="Toggle Sidebar"><span class="icon"></span></button><!--[--><a class="route-link vp-brand" href="/dsong-lab.github.io/" aria-label="Take me home"><img class="vp-nav-logo" src="/dsong-lab.github.io/favicon.ico" alt><!----><span class="vp-site-name hide-in-pad">DSong Lab</span></a><!--]--></div><div class="vp-navbar-center"><!--[--><nav class="vp-nav-links"><div class="vp-nav-item hide-in-mobile"><a class="route-link route-link-active auto-link" href="/dsong-lab.github.io/research.html" aria-label="Research" iconsizing="height"><!---->Research<!----></a></div><div class="vp-nav-item hide-in-mobile"><a class="route-link auto-link" href="/dsong-lab.github.io/publications.html" aria-label="Publications" iconsizing="height"><!---->Publications<!----></a></div><div class="vp-nav-item hide-in-mobile"><a class="route-link auto-link" href="/dsong-lab.github.io/Team/" aria-label="Team" iconsizing="height"><!---->Team<!----></a></div><div class="vp-nav-item hide-in-mobile"><a class="auto-link external-link" href="https://theme-hope.vuejs.press/" aria-label="V2 Docs" rel="noopener noreferrer" target="_blank" iconsizing="height"><!--[--><iconify-icon class="vp-icon" icon="fa6-solid:book" height="1em" sizing="height"></iconify-icon><!--]-->V2 Docs<!----></a></div></nav><!--]--></div><div class="vp-navbar-end"><!--[--><!----><div class="vp-nav-item hide-in-mobile"><button type="button" class="vp-color-mode-switch" id="color-mode-switch"><svg xmlns="http://www.w3.org/2000/svg" class="icon auto-icon" viewBox="0 0 1024 1024" fill="currentColor" aria-label="auto icon" name="auto" style="display:block;"><path d="M512 992C246.92 992 32 777.08 32 512S246.92 32 512 32s480 214.92 480 480-214.92 480-480 480zm0-840c-198.78 0-360 161.22-360 360 0 198.84 161.22 360 360 360s360-161.16 360-360c0-198.78-161.22-360-360-360zm0 660V212c165.72 0 300 134.34 300 300 0 165.72-134.28 300-300 300z"></path></svg><svg xmlns="http://www.w3.org/2000/svg" class="icon dark-icon" viewBox="0 0 1024 1024" fill="currentColor" aria-label="dark icon" name="dark" style="display:none;"><path d="M524.8 938.667h-4.267a439.893 439.893 0 0 1-313.173-134.4 446.293 446.293 0 0 1-11.093-597.334A432.213 432.213 0 0 1 366.933 90.027a42.667 42.667 0 0 1 45.227 9.386 42.667 42.667 0 0 1 10.24 42.667 358.4 358.4 0 0 0 82.773 375.893 361.387 361.387 0 0 0 376.747 82.774 42.667 42.667 0 0 1 54.187 55.04 433.493 433.493 0 0 1-99.84 154.88 438.613 438.613 0 0 1-311.467 128z"></path></svg><svg xmlns="http://www.w3.org/2000/svg" class="icon light-icon" viewBox="0 0 1024 1024" fill="currentColor" aria-label="light icon" name="light" style="display:none;"><path d="M952 552h-80a40 40 0 0 1 0-80h80a40 40 0 0 1 0 80zM801.88 280.08a41 41 0 0 1-57.96-57.96l57.96-58a41.04 41.04 0 0 1 58 58l-58 57.96zM512 752a240 240 0 1 1 0-480 240 240 0 0 1 0 480zm0-560a40 40 0 0 1-40-40V72a40 40 0 0 1 80 0v80a40 40 0 0 1-40 40zm-289.88 88.08-58-57.96a41.04 41.04 0 0 1 58-58l57.96 58a41 41 0 0 1-57.96 57.96zM192 512a40 40 0 0 1-40 40H72a40 40 0 0 1 0-80h80a40 40 0 0 1 40 40zm30.12 231.92a41 41 0 0 1 57.96 57.96l-57.96 58a41.04 41.04 0 0 1-58-58l58-57.96zM512 832a40 40 0 0 1 40 40v80a40 40 0 0 1-80 0v-80a40 40 0 0 1 40-40zm289.88-88.08 58 57.96a41.04 41.04 0 0 1-58 58l-57.96-58a41 41 0 0 1 57.96-57.96z"></path></svg></button></div><!--]--><button type="button" class="vp-toggle-navbar-button" aria-label="Toggle Navbar" aria-expanded="false" aria-controls="nav-screen"><span><span class="vp-top"></span><span class="vp-middle"></span><span class="vp-bottom"></span></span></button></div></header><!----><!--]--><!----><div class="toggle-sidebar-wrapper"><span class="arrow start"></span></div><aside id="sidebar" class="vp-sidebar" vp-sidebar><!----><ul class="vp-sidebar-links"></ul><!----></aside><!--[--><main id="main-content" class="vp-page"><!--[--><!----><!----><nav class="vp-breadcrumb disable"></nav><div class="vp-page-title"><h1><!---->Research</h1><!----><hr></div><!----><!----><div class="theme-hope-content" vp-content><p>Our lab studies various problems of data science in genomics, combining <em>bioinformatics</em>, <em>statistics</em>, and <em>machine learning</em>. We develop computational tools for analyzing &quot;omics&quot; data. Currently, our lab focuses on single-cell transcriptomics (gene expression within individual cells) and spatial transcriptomics (gene expression across spatial locations). One central goal is to better understand the cellular heterogenuity and distinguish the biological signals from techinical variation. Several ongoing research directions include:</p><h3 id="omics-data-simulation" tabindex="-1"><a class="header-anchor" href="#omics-data-simulation"><span>Omics data simulation</span></a></h3><div style="display:flex;align-items:center;"><div style="flex:1;"><p><em>How can we generate realistic omics data from computers?</em> The short answer is building a model that learns from the real data. The realistic simulation of single-cell and spatial multi-omics data plays a critical role in both evaluating <strong>performance of computational tools</strong> and facilitating the exploration of <strong>experimental designs</strong>. We developed <a href="https://doi.org/10.1038/s41587-023-01772-1" target="_blank" rel="noopener noreferrer">scDesign3 (Nature Biotechnology, 2024)</a>, the first an &quot;all-in-one&quot; multimodal single-cell and spatial omics simulator which uses a parametric model. I also contributed to the development of <a href="https://doi.org/10.1038/s41467-023-43162-w" target="_blank" rel="noopener noreferrer">scReadSim (Nature Communications, 2023)</a>, a single-cell RNA-seq and ATAC-seq sequencing read read simulator, and <a href="https://doi.org/10.1186/s13059-021-02367-2" target="_blank" rel="noopener noreferrer">scDesign2 (Genome Biology, 2021)</a>, the predecessor of scDesign3.</p></div><div style="margin-left:10px;"><img src="https://dsong-lab.github.io/images/research/scDesign3_example.png" alt="Description" width="300px"></div></div><h3 id="statistical-testing-of-gene-expression-and-co-expression" tabindex="-1"><a class="header-anchor" href="#statistical-testing-of-gene-expression-and-co-expression"><span>Statistical testing of gene expression and co-expression</span></a></h3><div style="display:flex;align-items:center;"><div style="flex:1;"><p><em>How can we accurately detect the changes in gene expression level and co-expression strengh?</em> We need to consider all types if variation and randomess for a solid conclusion. For example, in scRNA-seq, the cell types are usually annotated by clustering, then the differential expression (DE) test between clusters will find many fake DE genes (<strong>double dipping</strong>). We developed <a href="https://www.biorxiv.org/content/10.1101/2023.07.21.550107v1" target="_blank" rel="noopener noreferrer">ClusterDE (Accepted by RECOMB2025)</a>, a post-clustering DE test method controlling false discovery rate (FDR) under double dipping. Another exmample is the trajectory (pseudotime) inference; people usually ignore the fact that pseudotime is <strong>random</strong>. I developed <a href="https://doi.org/10.1186/s13059-021-02341-y" target="_blank" rel="noopener noreferrer">PseudotimeDE (Genome Biology, 2021)</a>, a DE test method for checking gene changes along cell pseudotime accounting for the uncertainty of pseudotime. In addition, we developed <a href="https://doi.org/10.1093/bioinformatics/btac423" target="_blank" rel="noopener noreferrer">scGTM (Bioinformatics, 2022)</a>, an interpretable statistical model for capturing <strong>a gene&#39;s expression trend</strong>.<br> Recently, we are developing a novel co-expression modeling method, named <a href="https://github.com/chexjiang/spCorr" target="_blank" rel="noopener noreferrer">spCorr</a>, for detecting the <strong>dynamic co-expression changes across different locations</strong> in spatial transcriptomics. By using a novel regression model, spCorr is very flexible and can deal with various confounding effects.</p></div><div style="margin-left:10px;"><img src="https://dsong-lab.github.io/images/research/ClusterDE_cartoon.png" alt="Description" width="300px"></div></div><h3 id="reducing-the-complexity-of-large-scale-omics-data" tabindex="-1"><a class="header-anchor" href="#reducing-the-complexity-of-large-scale-omics-data"><span>Reducing the complexity of large-scale omics data</span></a></h3><div style="display:flex;align-items:center;"><div style="flex:1;"><p><em>How can we reduce the complexity of current datasets with numerous genes and millions of cells?</em> In the era of big data, one simple solution is selecting the <strong>most imformative (or representitive) genes and cells</strong>. We developed <a href="https://doi.org/10.1093/bioinformatics/btab273" target="_blank" rel="noopener noreferrer">scPNMF (Bioinformatics; ISMB/ECCB, 2021)</a>, a gene selection method by using dimentionality reduction (Projective NMF) to get a sparse encoding of the expression matrix. In addition, we developed <a href="https://doi.org/10.1093/bioinformatics/btac271" target="_blank" rel="noopener noreferrer">scSampler (Bioinformatics, 2022)</a>, a diveristy-preserving cell subsampling method for large-scale datasets.</p></div><div style="margin-left:10px;"><img src="https://dsong-lab.github.io/images/research/scSampler_example.png" alt="Description" width="300px"></div></div></div><!----><footer class="vp-page-meta"><!----><div class="vp-meta-item git-info"><div class="update-time"><span class="vp-meta-label">Last update: </span><span class="vp-meta-info" data-allow-mismatch="text">3/5/2025, 3:41:45 PM</span></div><div class="contributors"><span class="vp-meta-label">Contributors: </span><!--[--><!--[--><span class="vp-meta-info" title="email: 21301010003@m.fudan.edu.cn">CZHanoi</span><!--]--><!--]--></div></div></footer><!----><!----><!----><!--]--></main><!--]--><!----></div><!--]--><!--]--><!--[--><!----><!--[--><!--]--><!--]--><!--]--></div>
    <script src="/dsong-lab.github.io/assets/js/runtime~app.267cafa3.js" defer></script><script src="/dsong-lab.github.io/assets/js/780.a6b3df29.js" defer></script><script src="/dsong-lab.github.io/assets/js/app.9e8e9631.js" defer></script>
  </body>
</html>
