---
title: "Publications"
---

## Preprint

1. Hemminger, Sanchez-Tam, De Ocampo, Wang, Underwood, Xie, Zhao, **Song**, Li, Dong, and Wollman.  
   "Spatial Single-Cell Mapping of Transcriptional Differences Across Genetic Backgrounds in Mouse Brains."  
   *bioRxiv* (2024).  
   DOI: [10.1101/2024.10.08.617260](https://doi.org/10.1101/2024.10.08.617260)

2. Li, M.Patel, **Song**, Yan, Li, and Pinello.  
   "Benchmarking computational methods to identify spatially variable genes and peaks."  
   *bioRxiv* (2023).  
   DOI: [10.1101/2023.12.02.569717](https://doi.org/10.1101/2023.12.02.569717)

3. Song, Liu, Dai, McMyn, Wang, Yang, Krejci, Vasilyev, Untermoser, Loregger, **Song**, Williams, Rosen, Cheng, Chao, Kale, Zhang, Diao, Bürckstümmer, Siliciano, Li, Siliciano, Huangfu, and Li.  
   "Decoding Heterogenous Single-cell Perturbation Responses."  
   *bioRxiv* (2023).  
   DOI: [10.1101/2023.10.30.564796](https://doi.org/10.1101/2023.10.30.564796)

4. Wang, Zhai, **Song**, and Li.  
   "Review of computational methods for estimating cell potency from single-cell RNA-seq data, with a detailed analysis of discrepancies between method description and code implementation."  
   *arXiv* (2023).  
   DOI: [10.48550/arXiv.2309.13518](https://doi.org/10.48550/arXiv.2309.13518)

5. **Song**, Li, Ge, and Li.  
   "ClusterDE: a post-clustering differential expression (DE) method robust to false-positive inflation caused by double dipping."  
   *bioRxiv* (2023).  
   DOI: [10.1101/2023.07.21.550107](https://doi.org/10.1101/2023.07.21.550107)


## Peer reviewed

1. Kock, Kimes, Gisselbrecht, Inukai, Phanor, Anderson, Ramakrishnan, Lipper, **Song**, Kurland, Rogers, Jeong, Blacklow, Irizarry, and Bulyk.  
   "DNA binding analysis of rare variants in homeodomains reveals homeodomain specificity-determining residues."  
   *Nature Communications* (2024).  
   DOI: [10.1038/s41467-024-47396-0](https://doi.org/10.1038/s41467-024-47396-0)

2. **Song**, Wang, Yan, Liu, Sun, and Li.  
   "scDesign3 generates realistic in silico data for multimodal single-cell and spatial omics."  
   *Nature Biotechnology* (2024).  
   DOI: [10.1038/s41587-023-01772-1](https://doi.org/10.1038/s41587-023-01772-1)

3. Yan, **Song**, and Li.  
   "scReadSim: a single-cell RNA-seq and ATAC-seq read simulator."  
   *Nature Communications* (2023).  
   DOI: [10.1038/s41467-023-43162-w](https://doi.org/10.1038/s41467-023-43162-w)

4. Cui, **Song**, Wong, and Li.  
   "Single-cell generalized trend model (scGTM): a flexible and interpretable model of gene expression trend along cell pseudotime."  
   *Bioinformatics* (2022).  
   DOI: [10.1093/bioinformatics/btac423](https://doi.org/10.1093/bioinformatics/btac423)

5. Jiang, Sun, **Song**, and Li.  
   "Statistics or biology: the zero-inflation controversy about scRNA-seq data."  
   *Genome Biology* (2022).  
   DOI: [10.1186/s13059-022-02601-5](https://doi.org/10.1186/s13059-022-02601-5)

6. **Song**, Xi, Li, and Wang.  
   "scSampler: fast diversity-preserving subsampling of large-scale single-cell transcriptomic data."  
   *Bioinformatics* (2022).  
   DOI: [10.1093/bioinformatics/btac271](https://doi.org/10.1093/bioinformatics/btac271)

7. Sun, **Song**, Li, and Li.  
   "Simulating Single-Cell Gene Expression Count Data with Preserved Gene Correlations by scDesign2."  
   *Journal of Computational Biology* (2022).  
   DOI: [10.1089/cmb.2021.0440](https://doi.org/10.1089/cmb.2021.0440)

8. Ge, Chen, **Song**, McDermott, Woyshner, Manousopoulou, Wang, Li, Wang, and Li.  
   "Clipper: p-value-free FDR control on high-throughput data from two conditions."  
   *Genome Biology* (2021).  
   DOI: [10.1186/s13059-021-02506-9](https://doi.org/10.1186/s13059-021-02506-9)

9. **Song**, Li, Hemminger, Wollman, and Li.  
   "scPNMF: sparse gene encoding of single cells to facilitate gene selection for targeted gene profiling."  
   *Bioinformatics* (2021).  
   DOI: [10.1093/bioinformatics/btab273](https://doi.org/10.1093/bioinformatics/btab273)

10. Sun, **Song**, Li, and Li.  
    "scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured."  
    *Genome Biology* (2021).  
    DOI: [10.1186/s13059-021-02367-2](https://doi.org/10.1186/s13059-021-02367-2)

11. **Song** and Li.  
    "PseudotimeDE: inference of differential gene expression along cell pseudotime with well-calibrated p-values from single-cell RNA sequencing data."  
    *Genome Biology* (2021).  
    DOI: [10.1186/s13059-021-02341-y](https://doi.org/10.1186/s13059-021-02341-y)

12. Miller, Hayashi, **Song**, and Wiens.  
    "Explaining the ocean's richest biodiversity hotspot and global patterns of fish diversity."  
    *Proceedings of the Royal Society B: Biological Sciences* (2018).  
    DOI: [10.1098/rspb.2018.1314](https://doi.org/10.1098/rspb.2018.1314)

13. **Song**, Wang, Song, Zhou, Xu, Yang, Yang, and Lu.  
    "Increased novel single nucleotide polymorphisms in weedy rice populations associated with the change of farming styles: Implications in adaptive mutation and evolution."  
    *Journal of Systematics and Evolution* (2017).  
    DOI: [10.1111/jse.12230](https://doi.org/10.1111/jse.12230)
